modules/ROOT/pages/configuration/server/occ_command.adoc

```diff
@@ -1295,6 +1295,71 @@ files_external:create [options] [--] <mount_point> <storage_backend> <authentica
 Each Storage Backend needs its corresponding authentication methods.
 ====
 
+.. _files_external_create_label:
```

Comment:
Pls change this to a valid [AsciiDoc anchor](https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#links).